kong/runloop/balancer.lua

```diff
@@ -217,9 +217,9 @@ do
 
     ------------------------------------------------------------------------------
     -- Callback function that informs the healthchecker when targets are added
-    -- or removed to a balancer.
+    -- or removed to a balancer and when targets health status change.
     -- @param balancer the ring balancer object that triggers this callback.
-    -- @param action "added" or "removed"
+    -- @param action "added", "removed", or "health"
```

Comment:
Is the "health" option here being handled just for logging purposes? I believe lua-resty-healthcheck already logs health changes, so these would produce duplicated log lines. If this was done just to avoid the warning in line 252, I'd consume the `"health"` actions and do nothing with it, here, or lower them to DEBUG in case we're concerned that the lua-resty-healthcheck logs and balancer responses don't match.

Reply:
Log message lowered to DEBUG.